src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -289,6 +290,114 @@ public static String composeRpcDefaultMethodHeaderSampleCode(
             .build());
   }
 
+  public static String composeLroCallableMethodHeaderSampleCode(
+      Method method,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes) {
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+    // Assign method's request variable with the default value.
+    VariableExpr requestVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("request").setType(method.inputType()).build());
+    Message requestMessage = messageTypes.get(method.inputType().reference().simpleName());
+    Preconditions.checkNotNull(requestMessage);
+    Expr requestBuilderExpr =
+        DefaultValueComposer.createSimpleMessageBuilderExpr(
+            requestMessage, resourceNames, messageTypes);
+    AssignmentExpr requestAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(requestVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(requestBuilderExpr)
+            .build();
+
+    List<Expr> bodyExprs = new ArrayList<>();
+    bodyExprs.add(requestAssignmentExpr);
+
+    // Create OperationFuture variable expr with invoking client's lro callable method.
+    // e.g. OperationFuture<Empty, WaitMetadata> future =
+    //          echoClient.waitOperationCallable().futureCall(request);
+    TypeNode operationFutureType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(OperationFuture.class)
+                .setGenerics(
+                    method.lro().responseType().reference(),
+                    method.lro().metadataType().reference())
+                .build());
+    VariableExpr operationFutureVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("future").setType(operationFutureType).build());
+    MethodInvocationExpr rpcMethodInvocationExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(clientVarExpr)
+            .setMethodName(
+                String.format("%sOperationCallable", JavaStyle.toLowerCamelCase(method.name())))
+            .build();
+    rpcMethodInvocationExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(rpcMethodInvocationExpr)
+            .setMethodName("futureCall")
+            .setArguments(requestVarExpr)
+            .setReturnType(operationFutureType)
+            .build();
+    bodyExprs.add(
+        AssignmentExpr.builder()
+            .setVariableExpr(operationFutureVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(rpcMethodInvocationExpr)
+            .build());
+
+    List<Statement> bodyStatements =
```

Comment:
What's the motivation for adding in the bodyExprs incrementally, rather than all at once at the end?

Reply:
Not sure if I am understanding here. 
Do you mean that instead of bodyExprs.add(..exprs) one-by-one follow up the expr declaration separately, we should add all exprs in one place, like
```
bodyExprs.add(requestAssignmentExpr);
bodyExprs.add(rpcMethodArgAssignmentExpr);
...
```

Or ask for why make two times transfer from `bodyExprs` -> `bodyStatements`. If this is what are you looking for, then the logic I am think here is inserting a `StatementComment` among the `bodyExprs`.
```
// define requestAssignmentExpr
body.Exprs(requestAssignmentExpr);
// define rpcMethodArgAssignmentExpr
bodyExprs.add(rpcMethodArgAssignmentExpr);

bodyExprs -> bodyStatements

bodyStatements.add(CommentStatement.withComment(LineComment.withComment("Do something;"))

bodyExprs.clear();
bodyExprs.add(...Exprs)

bodyStatements.addAll(..expr -> ExpressionStatement)
```

Reply:
The second case, thanks for clarifying.